app/spec.html

```diff
@@ -9,6 +9,15 @@
     <script src="vendor/jasmine/jasmine.js"></script>
     <script src="vendor/jasmine/jasmine-html.js"></script>
     <script src="vendor/jasmine/boot.js"></script>
+    <script>
+        var dnode = require('dnode');
+        var ClientReporter = require('./spec_helpers/client_reporter');
+        var clientReporter = new ClientReporter();
+        jasmine.getEnv().addReporter(clientReporter);
+        dnode.connect(5004).on('remote', function(remote) {
+            clientReporter.setEmitter(remote);
```

Comment:
This sets the emitter -after- it's created so jasmine can have the object now. Jasmine is started in the `boot.js` file listed earlier. Adding the `spec.js` file after doesn't work.